src/fswAlgorithms/attGuidance/sunSearch/sunSearch.cpp

```diff
@@ -106,6 +142,33 @@ void SunSearch::ComputeKinematicProperties(int const index)
 }
 
 
+/*! Define this method to compute the rate and acceleration as function of time
+    @return void
+    */
+void SunSearch::ComputeReferenceMotion(uint64_t const CurrentSimNanos, int const index, double *omega_RN, double *domega_RN)
+{
+    double zeroTime = 0;
+    for (int i=0; i<index; ++i) {
+        zeroTime += this->slewTotalTime[i];
+    }
+    double LocalSimSeconds = (CurrentSimNanos - this->resetTime) * NANO2SEC - zeroTime;
```

Comment:
:large_orange_diamond: These time variables can all be lower case `localSimSeconds`, `currentSimNanos`

Reply:
Done